src/main/java/org/folio/circulation/resources/RequestCirculationRulesEngineResource.java

```diff
@@ -0,0 +1,79 @@
+package org.folio.circulation.resources;
+
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.RoutingContext;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.folio.circulation.rules.Drools;
+import org.folio.circulation.support.OkJsonHttpResult;
+
+import static org.folio.circulation.support.http.server.ServerErrorResponse.internalError;
+
+/**
+ * The circulation rules engine calculates the request policy based on
+ * item type, request type, patron type and shelving location.
+ */
+public class RequestCirculationRulesEngineResource extends AbstractCirculationRulesEngineResource {
```

Comment:
Could you update this javaDoc? What this particular class does?

Reply:
@Anatolii-Starkov Please can you expand upon what you are expecting? We typically don't provide JavaDoc documentation with most of the module code.